README.md

```diff
@@ -35,8 +36,7 @@ massively useful. To set your own choice of column, add this to Foreman's config
 ```
 
 `title` is an arbitrary string which is displayed as the column header. `content` is
-a method call to the `Host` object, using `host.send`. You can also access `Hash` objects
-as well:
+a method call to the `Host` object, using `host.send`. You can also access `Hash` or `Params` objects:
```

Comment:
This is good to merge - but for the record, this works becuase "params" is just another method call on Host which returns a hash - just like "facts_hash" :)